### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
Screen Shot 2022-07-19 at 10 38 33 AM

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts",
+        "cypress.config.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (NicolasB2) [inline tsconfig.json:19]: EOF

File: cypress/support/e2e.ts [hunk 2]
```diff
@@ -17,4 +17,4 @@
 import './commands'
 
 // Alternatively you can use CommonJS syntax:
-// require('./commands')
+// require('./commands')
```
Comment (NicolasB2) [inline cypress/support/e2e.ts:20]: EOF

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -22,4 +22,4 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (NicolasB2) [inline cypress/support/commands.ts:25]: EOF

File: tsconfig.json (hunk 1 above)
Comment (SebastianGomezVidal) [reply tsconfig.json]: Hello Nicolas, sorry about that but I think  I am gonna get a lot of these type of errors in the future, I am working on a mac and Although I am putting this EOF sometimes in visual code the change is not captured, I will see what else can I do about it. Thanks

Screen Shot 2022-07-19 at 10 38 33 AM

Comment (SebastianGomezVidal) [reply tsconfig.json]: Screen Shot 2022-07-19 at 11 11 56 AM

**Pull request merged**